shared/main.desktop.js

```diff
@@ -53,6 +55,9 @@ class Main extends Component<void, Props, void> {
   componentDidMount () {
     if (flags.admin) window.addEventListener('keydown', this._handleKeyDown)
     ipcRenderer.send('showTray', this.props.menuBadge)
+
+    window.addEventListener('online', this.props.checkReachability)
```

Comment:
Created issue to deal with this: https://keybase.atlassian.net/browse/DESKTOP-2358